### PR TITLE
Gh-actions-refactor

### PR DESCRIPTION
**Pull request by gpmayorga**
## Motivation
- New release process and pipeline:
  - [Specs and release process](https://centrifuge.hackmd.io/MFsnRldyQSa4cadx11OtVg?view) (Private)
  - Issue #1074 
- Secure our deployments. Issue #1075 
- Move out of Netlify hosting

## Features
- New GHA pipelines for all of  the apps in this monorepo
- Serve sites with Google and Cloudflare
- Secure PROD deployments
- Trim down Github secrets to the minimum
- Multi-env strategy deployments w/ tags and releases
(see more in the links above)

### CODE REVIEW

**Comment by gpmayorga** [comment]
reopening to test

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+name: 'NPM publish (manual)'
+on:
+  workflow_dispatch:
+    inputs:
+      app_name:
+        description: "Choose an app to publish to NPM"
+        type: choice
```
Comment (onnovisser) [inline .github/workflows/npm-publish.yml:7]: Can we also choose a tag here, or will it always publish the latest from `main` ?

Comment (gpmayorga) [reply .github/workflows/npm-publish.yml:7]: So far this is only triggered when someone clicks on "Run workflow" through the UI. When you click on that, it'll prompt you to select a branch or a tag. 
Right now it's free for all, I have not implemented any restrictions anywhere so you can virtually publish from any commit. 
If we want to enforce some guidelines on what should and should not be published using code we could add it to this file later.

File: centrifuge-app/.env-config/.env.demo [hunk 2]
```diff
@@ -0,0 +1,15 @@
+REACT_APP_COLLATOR_WSS_URL=wss://fullnode.demo.cntrfg.com
+REACT_APP_DEFAULT_NODE_URL=
+REACT_APP_DEFAULT_UNLIST_POOLS=
```
Comment (Hornebom) [inline centrifuge-app/.env-config/.env.demo:3]: REACT_APP_DEFAULT_UNLIST_POOLS=true

File: centrifuge-app/.env-config/.env.catalyst [hunk 3]
```diff
@@ -0,0 +1,15 @@
+REACT_APP_COLLATOR_WSS_URL=wss://fullnode.catalyst.cntrfg.com
+REACT_APP_DEFAULT_NODE_URL=
+REACT_APP_DEFAULT_UNLIST_POOLS=
```
Comment (Hornebom) [inline centrifuge-app/.env-config/.env.catalyst:3]: REACT_APP_DEFAULT_UNLIST_POOLS=false

File: centrifuge-app/.env-config/.env.altair [hunk 4]
```diff
@@ -0,0 +1,15 @@
+REACT_APP_COLLATOR_WSS_URL=wss://fullnode.altair.centrifuge.io
+REACT_APP_DEFAULT_NODE_URL=
+REACT_APP_DEFAULT_UNLIST_POOLS=
```
Comment (Hornebom) [inline centrifuge-app/.env-config/.env.altair:3]: REACT_APP_DEFAULT_UNLIST_POOLS=false

File: centrifuge-app/.env-config/.env.development [hunk 5]
```diff
@@ -0,0 +1,15 @@
+REACT_APP_COLLATOR_WSS_URL=wss://fullnode.development.cntrfg.com
+REACT_APP_DEFAULT_NODE_URL=
+REACT_APP_DEFAULT_UNLIST_POOLS=
```
Comment (Hornebom) [inline centrifuge-app/.env-config/.env.development:3]: REACT_APP_DEFAULT_UNLIST_POOLS=false

File: centrifuge-app/.env-config/.env.production [hunk 6]
```diff
@@ -0,0 +1,15 @@
+REACT_APP_COLLATOR_WSS_URL=wss://fullnode.parachain.centrifuge.io
+REACT_APP_DEFAULT_NODE_URL=''
+REACT_APP_DEFAULT_UNLIST_POOLS=
```
Comment (Hornebom) [inline centrifuge-app/.env-config/.env.production:3]: REACT_APP_DEFAULT_UNLIST_POOLS=false

**Comment by gpmayorga** [comment]
I've enabled auto-merge after re-testing pretty much the whole pipeline with some modifications to fix staging deployments, this is ready to go and be merged into any other PR now

**Review by sophialittlejohn — APPROVED** [review]
LFG

**Pull request merged**